pom.xml

```diff
@@ -374,7 +374,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hadoop</groupId>
                 <artifactId>hadoop-apache2</artifactId>
-                <version>2.7.4-4</version>
+                <version>2.7.4-5-SNAPSHOT</version>
```

Comment:
Could you make a new release of `hadoop-apache2`

Reply:
@electrum David will help make a release of hadoop-apache2. Thanks David!